modules/hal_nxp/mcux/mcux-sdk-ng/drivers/drivers.cmake

```diff
@@ -160,6 +160,9 @@ elseif(CONFIG_SOC_MIMX9596)
   set_variable_ifdef(CONFIG_ETH_NXP_IMX_NETC CONFIG_MCUX_COMPONENT_driver.netc_imx95)
   set_variable_ifdef(CONFIG_ETH_NXP_IMX_NETC CONFIG_MCUX_COMPONENT_driver.msgintr)
   set_variable_ifdef(CONFIG_ETH_NXP_IMX_NETC CONFIG_MCUX_COMPONENT_driver.irqsteer)
+elseif(CONFIG_SOC_MIMX94398)
+  set_variable_ifdef(CONFIG_ETH_NXP_IMX_NETC CONFIG_MCUX_COMPONENT_driver.netc_imx943)
+  set_variable_ifdef(CONFIG_ETH_NXP_IMX_NETC CONFIG_MCUX_COMPONENT_driver.msgintr)
```

Comment:
Cortex-A Core will not use irqsteer and msgintr, so how about to refine these code like:
```
if(CONFIG_ETH_NXP_IMX_NETC)
set_variable_ifdef(CONFIG_SOC_MIMXRT1189 CONFIG_MCUX_COMPONENT_driver.netc_rt1180)
set_variable_ifdef(CONFIG_SOC_MIMX9596 CONFIG_MCUX_COMPONENT_driver.netc_imx95)
set_variable_ifdef(CONFIG_SOC_MIMX94398 CONFIG_MCUX_COMPONENT_driver.netc_imx943)

if(CONFIG_SOC_MIMXRT1189 OR CONFIG_SOC_MIMX9596_M7 OR CONFIG_SOC_MIMX94398_M33)
  set(CONFIG_MCUX_COMPONENT_driver.msgintr ON)
endif()

set_variable_ifdef(CONFIG_SOC_MIMX9596_M7 CONFIG_MCUX_COMPONENT_driver.irqsteer)
endif()
```

Reply:
Thanks. Will update.